doc/guide/Guide_CLIMADA_conventions.ipynb

```diff
@@ -65,9 +83,194 @@
+    "Please note that pull requests will not be merged if these checks fails. The easiest way to ensure this, is to use [pre-commit hooks](../guide/Guide_Git_Development.ipynb#pre-commit-hooks), which will allow you to both run the checks and apply fixes when creating a new commit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": true,
     "slideshow": {
      "slide_type": "slide"
-    }
+    },
+    "tags": []
+   },
+   "source": [
+    "### `black`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": "slide"
+    },
+    "tags": []
+   },
+   "source": [
+    "We chose black as our formatter because it perfectly fits this need, quoting directly from the project\n",
+    "\n",
+    "> Black is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting. In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters.\n",
+    "Blackened code looks the same regardless of the project you're reading. Formatting becomes transparent after a while and you can focus on the content instead.\n",
+    "Black makes code review faster by producing the smallest diffs possible.\n",
+    "\n",
+    "`black` automatically reformats your Python code to conform to the PEP 8 style guide, among other guidelines. It takes care of various aspects, including:\n",
+    "\n",
+    "- Line Length: By default, it wraps lines to 88 characters, though this can be adjusted.\n",
+    "- Indentation: Ensures consistent use of 4 spaces for indentation.\n",
+    "- String Quotes: Converts all strings to use double quotes by default.\n",
+    "- Spacing: Adjusts spacing around operators and after commas to maintain readability.\n",
+    "\n",
+    "For installation and more in depth information on black refer to [its documentation](https://black.readthedocs.io/en/stable/getting_started.html).\n",
+    "\n",
+    "Also note the [vscode plugin](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter) or [this post](https://stackoverflow.com/a/66458706/4703808) on how to use it with `spyder`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": "slide"
+    },
+    "tags": []
+   },
+   "source": [
+    "### `isort`\n",
+    "\n",
+    "`isort` is a Python utility / library to sort imports alphabetically, and automatically separated into sections and by type.\n",
+    "\n",
+    "Just like `black` it ensure consistency of the code, focusing on the imports\n",
+    "\n",
+    "For installation and more in depth information on `isort` refer to [its documentation](https://pycqa.github.io/isort/).\n",
+    "\n",
+    "Also note the [vscode plugin](https://marketplace.visualstudio.com/items?itemName=ms-python.isort)."
+   ]
+  },
```

Comment:
Do we also need to install `isort` separately? Or is this already included in the `black` plugin?